scene/gui/line_edit.cpp

```diff
@@ -265,26 +334,37 @@ void LineEdit::gui_input(const Ref<InputEvent> &p_event) {
 
 	Ref<InputEventMouseButton> b = p_event;
 
-	if (b.is_valid()) {
-		if (ime_text.length() != 0) {
-			// Ignore mouse clicks in IME input mode.
-			return;
```

Comment:
What happened to this part?

Reply:
Combined both checks `if (b.is_valid() && ime_text.is_empty())` in one statement and returned back the comment.